sdk/search/Azure.Search.Documents/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 11.4.0-beta.6 (Unreleased)
+## 11.4.0-beta.6 (2022-02-08)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added `Unk` as an `OcrSkillLanguage` value. The values are used to set the default language code for the [OCR cognitive skill](https://docs.microsoft.com/azure/search/cognitive-search-skill-ocr).
```

Comment:
Is there a public API called `Unk`?  I'm not seeing it in this PR, but if it exists should it be called `Unknown` instead?

Reply:
Yes, this was [added recently](https://github.com/Azure/azure-sdk-for-net/pull/25812) to the SDK. It was missing from the REST spec as a supported value.

Reply:
We should chat about this one during standup today because I've got a few questions: partly how this relates to a `null` value for the extensible enum, partly the discoverability of `Unk` vs `Unknown` _(since I definitely didn't realize that when I approved the original pr :smile:)_, and partly a concern that https://iso639-3.sil.org/code/unk is already a language code _(though it's ISO 639-3 and I think all the other values here are just ISO 639-1)_.

Reply:
Sounds good.
[The documentation](https://docs.microsoft.com/en-us/azure/search/cognitive-search-skill-ocr#skill-parameters) says -
![image](https://user-images.githubusercontent.com/8271806/152065573-8b9debb7-128e-4c55-a949-bc445366c418.png)

We can switch to saying `Unknown` to make the meaning clear, we have to remember to do it for every instance going forward. Currently, other skills do not use 'unknown' as a don't-care-figure-it-out option.